mods/ra/rules/defaults.yaml

```diff
@@ -544,10 +544,10 @@
 		AirborneCondition: airborne
 	Targetable@GROUND:
 		RequiresCondition: !airborne
-		TargetTypes: Ground, Vehicle
+		TargetTypes: GroundActor, Vehicle
 	Targetable@AIRBORNE:
 		RequiresCondition: airborne
-		TargetTypes: Air
+		TargetTypes: Airborne
```

Comment:
Shouldn't this be `AirborneActor` for consistency?

Reply:
I guess you're right.